keps/sig-network/3458-remove-transient-node-predicates-from-service-controller/kep.yaml

```diff
@@ -0,0 +1,24 @@
+prnumber: "3460"
+name: 3458-remove-transient-node-predicates-from-service-controller
+title: Remove transient node predicates from KCCM's service controller
+kep-number: "3458"
+authors: ['@alexanderConstantinescu']
+owning-sig: sig-network
+participating-sigs: [sig-network]
+reviewers:
+  - "@thockin"
+approvers:
+  - "@thockin"
+creation-date: "2022-08-07"
+last-updated: v1.27
+status: implementeable
+stage: stable
```

Comment:
I was wondering why it passed validation without the PRR file - and it's because of this one.
This should be beta  - and that will force validation in 1.27 release (not in 1.29).

BTW - I'm happy to take a PRR review for it.

Reply:
Dohh, I updated milestones yesterday, but forgot this section. Give me one hour and I will file a follow up PR with the PRR file, update to this section and a short section on https://github.com/kubernetes/enhancements/pull/3460/files#r1095489657

Reply:
Ref: https://github.com/kubernetes/enhancements/pull/3835